crates/engraver/src/render/ir/element.rs

```diff
@@ -6,6 +6,10 @@ pub enum Element<T> {
     Polygon(Polygon<T>),
     Symbol(Symbol<T>),
     Text(Text<T>),
+    Group {
+        id: Option<String>,
+        elements: Vec<Element<T>>,
+    },
```

Comment:
Could you make a `Group` struct so this can be `Group(Group<T>)` just to be consistent with the other cases in this enum?